docker/docker-compose-dist.yml

```diff
@@ -0,0 +1,36 @@
+#
+# The contents of this file are subject to the license and copyright
+# detailed in the LICENSE and NOTICE files at the root of the source
+# tree and available online at
+#
+# http://www.dspace.org/license/
+#
+
+# Docker Compose for running the DSpace Angular UI dist build
+# for previewing with the DSpace Demo site backend
+version: '3.7'
+networks:
+  dspacenet:
+services:
+  dspace-angular:
+    container_name: dspace-angular
+    environment:
+      DSPACE_UI_SSL: 'false'
+      DSPACE_UI_HOST: dspace-angular
+      DSPACE_UI_PORT: '4000'
+      DSPACE_UI_NAMESPACE: /
+      DSPACE_REST_SSL: 'true'
+      DSPACE_REST_HOST: api7.dspace.org
```

Comment:
Trying to decide if this is better to point at api7.dspace.org by default... or if it's better to just point at localhost:8080 (like `docker-compose.yml`).  While pointing it at api7.dspace.org means you don't need to run the backend via Docker... localhost:8080 would be more useful in testing a local "production-like" environment where everything is in Docker.

Not against the approach you took here... just thinking through which option is better to have by default.  I'm still undecided which approach I prefer.

Reply:
On this part iirc the SSR/CSR issue will be a problem (production mode in Docker referencing a localhost address) unless you get into some hosts file trickery? Re ISS: https://github.com/DSpace/dspace-angular/issues/1485 & PR https://github.com/DSpace/dspace-angular/pull/2069

Reply:
@mark-cooper : Good point.  I was thinking more of a very specific scenario... if you are running the `-dist` image on localhost:4000, then this *could* be set to localhost:8080.

However, that scenario may be rare... so, your point is well taken.  If the `-dist` image is running via any non-localhost URL, then having this pointing at localhost:8080 won't work. 

So, let's keep this as-is afterall.  It's the best setup we can do for now, and we can always change it later as needed.